src/test/java/com/snowplowanalytics/snowplow/tracker/emitter/BatchEmitterTest.java

```diff
@@ -60,14 +57,10 @@ public int get(TrackerPayload payload) {
         }
 
         @Override
-        public String getUrl() {
-            return null;
-        }
+        public String getUrl() { return null; }
```

Comment:
Just my opinion. I don't see this a coding style mistake. For me it's not that wrong to have the function on multiple lines even with a single line block.